doc/source/api/general/case_reader.rst

```diff
@@ -50,6 +50,35 @@ Along with basic functionality, the CaseFile class provides many additional feat
     >>> reader.rp_vars()
     >>> reader.config_vars()
 
+- **Extracts mesh data**
+  The CaseReader can be used to extract mesh data. This example shows how to
+  have the CaseFile class read a case file (.cas.h5) from the ``examples``
+  repository and extract and use mesh data:
+
+  .. code-block:: python
+
+      >>> from ansys.fluent.core import examples
+      >>> from ansys.fluent.core.filereader.casereader import CaseReader
+
+      >>> case_filepath = examples.download_file("elbow1.cas.h5", "pyfluent/file_session")
+      >>> reader = CaseReader(case_filepath=case_filepath)
+      >>> reader._mesh.get_surface_ids()
```

Comment:
For a public API, change to either `reader.mesh.get_surface_ids()` or `reader.get_surface_ids()`

Reply:
Done, Thank you. 'get_mesh()' was already available and has been used.